src/core/nginx.c

```diff
@@ -426,7 +426,7 @@ static void
 ngx_show_version_info(void)
 {
 #if (T_NGX_SERVER_INFO) 
-    ngx_write_stderr("Tengine version: " TENGINE_VER " (" NGINX_VER ")"
+    ngx_write_stderr("Tengine version: " TENGINE_VER_BUILD " (" NGINX_VER ")"
                      NGX_LINEFEED);
```

Comment:
-> 

```
ngx_write_stderr("Tengine version: " TENGINE_VER_BUILD  NGX_LINEFEED);
```

Reply:
`TENGINE_VER_BUILD` does not conatin `NGINX_VER`, current (and former) tengine outputs `Tengine version: Tengine/2.3.0 (nginx/1.15.9)` for `tengine -v`, omitting the `NGINX_VER` would be `Tengine version: Tengine/2.3.0` without the nginx part.

I think this could be a potential compatibility issue?

Reply:
Good catch!

I think outputing both version lines are good idea!

```
ngx_write_stderr("nginx version: " NGINX_VER_BUILD NGX_LINEFEED);

#if (T_NGX_SERVER_INFO)
ngx_write_stderr("Tengine version: " TENGINE_VER_BUILD  NGX_LINEFEED);
#endif

```

Reply:
Still, this makes the output format different from former releases. And this would print `NGX_BUILD` twice if it's set.

But considering of compatibility with vanilla nginx, it makes some senses for me, I'll push a commit later to fix this.